scripts/src/internal/jwt.ts

```diff
@@ -16,7 +16,7 @@ function getKeyForAlgorithm(alg: string): string {
 	return keyid;
 }
 
-export function sign(subject: string, payload: any, alg?: string) {
+export function sign(subject: string, payload: object, alg?: string) {
```

Comment:
@nitzantomer I made this change - reducing the amount of `any` in the code. Does this make sense?